test-suite/js/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/temporal/TemporalAccessorTest.scala

```diff
@@ -17,7 +17,7 @@ trait TemporalAccessorTest extends JasmineTest with ExpectExceptions {
     it("should respond to `range`") {
       for {
         accessor <- samples
-        field <- ChronoField.values()
+        field <- ChronoField.values() if field != ChronoField.DAY_OF_MONTH
```

Comment:
Fixed. Would be better to have the test in `LocalDateTest.scala`, but we need to add the exception here anyway and it's only temporarily anyway.